src/olympia/editors/tests/test_helpers.py

```diff
@@ -478,7 +479,27 @@ def test_notify_email(self):
             mail.outbox = []
             self.helper.handler.notify_email(template, 'Sample subject %s, %s')
             assert len(mail.outbox) == 1
-            assert mail.outbox[0].body, 'Expected a message'
+            assert mail.outbox[0].body
+
+    @override_switch('activity-email', active=True)
+    def test_notify_email_activity_email(self):
+        self.helper.set_data(self.get_data())
+        user = self.addon.listed_authors[0]
+        for template in ['nominated_to_nominated', 'nominated_to_preliminary',
+                         'nominated_to_public', 'nominated_to_sandbox',
```

Comment:
This whole test should imho just use `pytest.mark.parametrize` instead of relying on a for-loop, this makes debugging a failure much easier.